manifests/07-downloads-helm.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: helm-download-links
+spec:
+  description: |
+    Helm 3 is a package manager for Kubernetes applications which enables defining,
+    installing and upgrading applications packaged as Helm Charts.
```

Comment:
Oxford comma please :)

```suggestion
    installing, and upgrading applications packaged as Helm Charts.
```

Reply:
I think this comment was missed as well.